performance/include/IResourceComputingStrategy.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdint>
+#include "ISystemResourceEntity.hpp"
+
+namespace concord::performance {
+
+class IResourceComputingStrategy {
+ public:
+  virtual std::uint64_t computeAvailableResources(const ISystemResourceEntity &consensusMonitor,
+                                                  const ISystemResourceEntity &storageMonitor) = 0;
+};
+
+}  // namespace concord::performance
```

Comment:
Newline missing - Github shows it with this symbol.

Reply:
Done